Doc/library/datetime.rst

```diff
@@ -737,18 +737,16 @@ Instance methods:
 .. method:: date.strftime(format)
 
    Return a string representing the date, controlled by an explicit format string.
-   Format codes referring to hours, minutes or seconds will see 0 values. For a
-   complete list of formatting directives, see
-   :ref:`strftime-strptime-behavior`.
```

Comment:
This `ref` is converted to the title of the section: `strftime() and strptime() Behavior`.  The actual list of format codes is actually a subsection.  Adding a label to the subsection and updating the `ref` might result in a more readable sentence.